libraries/wasm-jit/Source/Logging/Logging.cpp

```diff
@@ -5,6 +5,10 @@
 #include <cstdarg>
 #include <cstdlib>
 
+namespace WASM
+{
+	bool check_limits = true;
+}
```

Comment:
Did you consider using an `inline static` of a struct instead? Seems like that would be cleaner than `extern`.

Reply:
I do like the idea of an `inline static` struct member, as it relies on a C++ feature that I feel should be more foolproof. Still since it is not my code, and this is a workaround for a specific compiler issue, I'd prefer to keep the change as minimal as possible.

Reply:
It seems a little strange putting something that has nothing to do with logging in the `Logging` library. Could it work in `IR` instead? The Logging library could easily be on the chopping block whenever someone has time to keep trimming down WAVM.

Reply:
Thanks Matt, it looks like it is working fine with the variable in `IR`. I'll push that change.